image/default_image.go

```diff
@@ -197,38 +200,49 @@ func (d DefaultImageService) Delete(imageName string) error {
 	if err != nil {
 		return err
 	}
-
+	// example ImageName : 7e2e51b85680d827fae08853dea32ad6:latest
+	// example ImageID :   7e2e51b85680d827fae08853dea32ad6
+	// https://github.com/alibaba/sealer/blob/f9d609c7fede47a7ac229bcd03d92dd0429b5038/image/reference/util.go#L59
 	imageMetadata, ok := imageMetadataMap[named.Raw()]
-	if !ok {
+	if !ok && strings.Contains(imageName, ":") {
 		return fmt.Errorf("failed to find image with name %s", imageName)
 	}
 
-	//1.untag image
-	err = imageutils.DeleteImage(imageName)
-	if err != nil {
-		return fmt.Errorf("failed to untag image %s, err: %s", imageName, err)
+	if strings.Contains(imageName, ":") {
+		//1.untag image
+		if err = imageutils.DeleteImage(imageName); err != nil {
+			return fmt.Errorf("failed to untag image %s, err: %w", imageName, err)
+		}
+		image, err = imageutils.GetImageByID(imageMetadata.ID)
```

Comment:
We should not ignore any errors

Reply:
> We should not ignore any errors
yes ,i,know ,but I should not have  ignore any errors.